### PR TITLE
fix: typo [release]

### PR DESCRIPTION
**Pull request by Sollimann**
fix typo

### CODE REVIEW

File: packages/wells/src/client/model/WellFilter.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export interface WellFilter {
    * @type {string[]}
    * @memberof WellFilter
    */
-  wellType?: string[];
+  wellTypes?: string[];
```
Comment (drsphelps) [inline packages/wells/src/client/model/WellFilter.ts:11]: I think this needs to be changed in the `WellFilterAPI` type as well

File: packages/wells/src/client/api/wellsApi.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export class WellsAPI extends ConfigureAPI {
       }
     }
     return {
-      wellType: filter.wellType,
+      wellType: filter.wellTypes,
```
Comment (sighol) [inline packages/wells/src/client/api/wellsApi.ts]: Should it be `wellTypes: ...` here?

**Review by sighol — APPROVED** [review]

File: packages/wells/src/client/api/wellsApi.ts (hunk 2 above)
Comment (Sollimann) [reply packages/wells/src/client/api/wellsApi.ts]: updated in `WellTypesApi` as well now

**Review by drsphelps — APPROVED** [review]
lgtm

**Pull request merged**